XfplEnvironment/XfplEnvironment.synproj

```diff
@@ -103,8 +103,7 @@
   <PropertyGroup>
     <PreBuildEvent>
     </PreBuildEvent>
-    <PostBuildEvent>
-    </PostBuildEvent>
+    <PostBuildEvent>copy /Y $(TargetPath) $(SolutionDir)$(ConfigurationName)\x64\$(TargetFileName)</PostBuildEvent>
```

Comment:
Why not just output to the right location instead?

Reply:
needs to be in both places